ocs_ci/ocs/ui/block_pool.py

```diff
@@ -342,3 +342,25 @@ def check_pool_compression_savings(self, pool_name):
             f"Pool name {pool_name} existence is {blockpool_compression_savings}"
         )
         return blockpool_compression_savings
+
+    def check_storage_class_attached(self, pool_name):
+        """
+        Checks the number of storage class attached to the pool.
+
+        Args:
+            pool_name (str): Name of the pool to check
+
+        Return:
+            int: The total number of storage classes attached to the pool.
+        """
+        self.navigate_block_pool_page()
+        self.do_click((f"a[data-test={pool_name}]", By.CSS_SELECTOR))
+        time.sleep(10)
```

Comment:
BaseUI.wait_for_element_attribute() should make the anticipation time shorter than 10s

Reply:
Here the test has to wait for the API results to be fetched to be reflected on the UI, the time.sleep is hardcoded here.